Gems/AtomTressFX/Code/Rendering/HairFeatureProcessor.h

```diff
@@ -217,7 +217,7 @@ namespace AZ
                 bool m_forceClearRenderData = false;
                 bool m_initialized = false;
                 bool m_isEnabled = true;
-                bool m_usePPLLRenderTechnique = true;
+                bool m_usePPLLRenderTechnique = false;
```

Comment:
Does this effect the main render pipeline for template used in new project (which uses MainRenderPipeline.pass defined in Atom Feature Common gem)?

Reply:
This should have no effect since it is being set in the constructor, but for consistency and verifying that by default ShortCut technique is set (due to memory usage), I set it here.